README.md

```diff
@@ -1,67 +1,20 @@
 ![commercetools logo](https://cdn.rawgit.com/commercetools/press-kit/master/PNG/72DPI/CT%20logo%20horizontal%20RGB%2072dpi.png)
 
-# Stock export
+## ⛔ [Deprecated], please use the [nodejs repo](https://github.com/commercetools/nodejs) instead
 
-[![NPM](https://nodei.co/npm/sphere-stock-export.png?downloads=true)](https://www.npmjs.org/package/sphere-stock-export)
-
-[![Build Status](https://secure.travis-ci.org/sphereio/sphere-stock-export.png?branch=master)](http://travis-ci.org/sphereio/sphere-stock-export) [![NPM version](https://badge.fury.io/js/sphere-stock-export.png)](http://badge.fury.io/js/sphere-stock-export) [![Coverage Status](https://coveralls.io/repos/sphereio/sphere-stock-export/badge.svg?branch=master)](https://coveralls.io/r/sphereio/sphere-stock-export?branch=master) [![Dependency Status](https://david-dm.org/sphereio/sphere-stock-export.svg)](https://david-dm.org/sphereio/sphere-stock-export) [![devDependency Status](https://david-dm.org/sphereio/sphere-stock-export/dev-status.svg)](https://david-dm.org/sphereio/sphere-stock-export#info=devDependencies)
+# Stock export (Inventory export)
 
 This module allows to export stocks to CSV.
 
-## Getting started
-
-```bash
-$ npm install -g sphere-stock-export
-
-# output help screen
-$ stock-export
-```
-
-## Usage
-
-General command line options can be seen by simply executing the command `node lib/run`.
-```
-node lib/run
-
-  Usage:     --projectKey <project-key> --clientId <client-id> --clientSecret <client-secret>
+This repo has been deprecated and the code migrated to this [repo](https://github.com/commercetools/nodejs).
```

Comment:
Don't we have any examples of usage? Shouldn't we help a user to migrate when we deprecate something?

Reply:
@LEQADA, I think the link to the documentation should be enough help, no?
`The documentation about this module can be found [here](https://commercetools.github.io/nodejs/cli/inventories-exporter.html)`

Reply:
Thanks for the link. I thought that those links are still old